src/test/fuzz/minisketch.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <minisketch.h>
+#include <node/minisketchwrapper.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <util/check.h>
+
+#include <map>
+#include <numeric>
+
+FUZZ_TARGET(minisketch)
+{
+    FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
+    const auto capacity{fuzzed_data_provider.ConsumeIntegralInRange<size_t>(1, 200)};
+    Minisketch sketch_a{Assert(MakeMinisketch32(capacity))};
+    Minisketch sketch_b{Assert(MakeMinisketch32(capacity))};
+
+    // Fill two sets and keep the difference in a map
+    std::map<uint32_t, bool> diff;
+    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 10000)
+    {
+        const auto entry{fuzzed_data_provider.ConsumeIntegralInRange<uint32_t>(1, std::numeric_limits<uint32_t>::max() - 1)};
+        const auto KeepDiff{[&] {
+            bool& mut{diff[entry]};
+            mut = !mut;
+        }};
+        CallOneOf(
+            fuzzed_data_provider,
+            [&] {
+                sketch_a.Add(entry);
+                KeepDiff();
+            },
+            [&] {
+                sketch_b.Add(entry);
+                KeepDiff();
+            },
+            [&] {
+                sketch_a.Add(entry);
+                sketch_b.Add(entry);
+            });
+    }
+    const auto num_diff{
+        std::accumulate(diff.begin(), diff.end(), size_t{0}, [](size_t n, const std::pair<const uint32_t, bool>& e) { return n + e.second; })};
```

Comment:
FWIW, you can use `[](auto n, const auto e&) { return n + e.second; }` as lambda too.

Reply:
Nice. I didn't know this was added in C++14.

Done.